project-base/kubernetes/deployments/rabbitmq.yml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    name: rabbitmq
+    labels:
+        app: rabbitmq
+spec:
+    replicas: 1
+    template:
+        metadata:
+            name: rabbitmq
+            labels:
+                app: rabbitmq
+        spec:
+            containers:
+                -   name: rabbitmq
+                    image: rabbitmq:3.7-management-alpine
+                    imagePullPolicy: IfNotPresent
+            restartPolicy: Always
```

Comment:
`Always` is the default restart policy [according to the docs](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy). I would omit it (just as we do in other manifests).

```suggestion
```

Reply:
Applied manually in bffa63ae9a2d508978602f29638afdc8be542656